stock_picking_delivery_info_computation/models/stock_picking.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Tecnativa - Victor M.M. Torres
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import api, models
+
+
+class StockPicking(models.Model):
+    _inherit = 'stock.picking'
+
+    @api.depends(
+        'product_id',
+        'move_lines',
+        'pack_operation_product_ids')
```

Comment:
You must add the subfields that influences in this compute:
```suggestion
        'pack_operation_product_ids',
        'pack_operation_product_ids.product_id',
        'pack_operation_product_ids.product_qty',
        'pack_operation_product_ids.qty_done',
    )
```